conf/storm.yaml.example

```diff
@@ -72,4 +72,11 @@
 #     argument:
 #       - endpoint: "metrics-collector.mycompany.org"
 #
-# storm.cluster.metrics.consumer.publish.interval.secs: 60
\ No newline at end of file
+# storm.cluster.metrics.consumer.publish.interval.secs: 60
+
+# Event Logger
+# topology.event.logger.register:
+#   - class: "org.apache.storm.metric.FileBasedEventLogger"
+#   - class: "org.mycompany.MyEventLogger"
+#     argument:
```

Comment:
Nit: May want to make this "arguments" if it is a list.

Reply:
I'm following the name for metrics consumer, but I think it is worth to fix. Moreover we're deprecating metrics consumer so it looks OK. I'll fix it.

Reply:
Ah also it is a `Map<String,Object>`. I think I should remove `-`. Will remove.